package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "studentproject-wecook",
+  "version": "1.0.0",
+  "description": "FACN4 WeCook student project",
+  "main": "index.js",
+  "scripts": {
+    "test": "NODE_ENV=test nyc tape testing/* | tap-spec",
+    "dev": "nodemon src/index.js",
+    "coverage": "nyc --reporter=lcov npm run test",
+    "start": "node src/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FACN4/StudentProject-WeCook.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/FACN4/StudentProject-WeCook/issues"
+  },
+  "homepage": "https://github.com/FACN4/StudentProject-WeCook#readme",
+  "devDependencies": {
+    "eslint": "^5.5.0",
+    "eslint-config-airbnb-base": "^13.1.0",
+    "eslint-plugin-import": "^2.14.0",
+    "jest": "^23.5.0",
+    "nodemon": "^1.18.4",
+    "nyc": "^13.0.1",
+    "supertest": "^3.3.0",
+    "tap-spec": "^5.0.0",
+    "tape": "^4.9.1",
+    "watch": "^1.0.2"
```

Comment:
Have a look at this https://stackoverflow.com/questions/25472665/watch-and-rerun-jest-js-tests. You may not need an extra watch module.